.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+- [ ] Ready for review
+- [ ] Follows [CONTRIBUTING](https://github.com/snyk/broker/blob/master/.github/CONTRIBUTING.md) rules
```

Comment:
Not sure that the broker repo template is the most fitting one. Try the templates of one of the cli plugins?